### PR TITLE
Update isoplot wrapper to vgalaxy1

### PR DESCRIPTION
**Pull request by llegregam**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by lecorguille — APPROVED** [review]

**Comment by lecorguille** [comment]
You still need a rebase :)
And an advice, avoid to work directly on your master branch.

Something like that (since you are working on your local master branch):

```bash
git add remote upstream git@github.com:workflow4metabolomics/tools-metabolomics.git
git fetch upstream
git checkout master
git pull upstream master
git push origin master
```

**Comment by llegregam** [comment]
Argh git still confuses my brain! Let me try again ^^

**Comment by llegregam** [comment]
Git tells me everything is up to date...

**Review by lecorguille — APPROVED** [review]

**Comment by lecorguille** [comment]
Replace by https://github.com/workflow4metabolomics/tools-metabolomics/pull/207 

FYI, I used `git cherry-pick 82c15fc8c1aede4c99f205c596768047ae0812dc` to just pick your last commit (using the commit tag) on a fresh branch from master

**Comment by llegregam** [comment]
Okey thank you for the info!
